jfrog_platform_cloud/README.md

```diff
@@ -0,0 +1,34 @@
+# Agent Check: JFrog Platform Cloud
+
+## Overview
+
+[JFrog](https://jfrog.com/) is a universal hybrid, end-to-end DevOps platform. JFrog Artifactory is the single solution for housing and managing all the artifacts, binaries, packages, files, containers, and components for use throughout your software supply chain.
+JFrog Artifactory serves as your central hub for DevOps, integrating with your tools and processes to improve automation, increase integrity, and incorporate best practices along the way.
+
+JFrog's SaaS Log Streamer is a log streaming solution built by JFrog for SaaS customers. This solution will stream JFrog Artifactory logs from the customer's JFrog SaaS instance straight into their Datadog instance.
+
+Customers who use both JFrog and Datadog will be able to visualize Artifactory logs inside pre-configured Datadog dashboards. This integration also has built-in support for Datadog log pipelines which means logs streamed from JFrog will be preprocessed and automatically converted into the Datadog log format, allowing teams to uniquely name logs per their needs, drill down into Artifactory logs through searchable facets, and easily monitor their JFrog SaaS instance.
+
+At the initial release this integration will stream the following artifactory logs to Datadog:
+
+- **access-audit.log**
+- **artifactory-request.log**
+- **access-security-audit.log**
+
+These logs will allow customers to readily know who accessed what repositories and how often. The logs will also show what IP addresses accessed those repositories. Log types such as traffic.log, artifactory-access.log and more request logs will be added to this integration in future updates.
+
+## Setup
+
+**Note:** The integration requires JFrog Enterprise Plus subscription. 
+
+### Installation
+
+Create [DataDog API key](https://app.datadoghq.com/organization-settings/api-keys), encrypt it with JFrog provided GPG key and save it to a text file.  
```

Comment:
Hey @danielmkn I thought this step was going to be changed so that the user did not have to encrypt/save their API key to a file?

Reply:
Oh, sorry, missed that one. Fixed now.